README.rst

```diff
@@ -50,14 +50,26 @@ The built documentation will be available at::
 
   /absolute/path/to/cellml-specification/build/singlehtml/index.html
 
+Building PDFs
+^^^^^^^^^^^^^
+
 To build the PDF form of the documentation, simply execute the following command::
 
   make latexpdf
 
-**Note:** creating the PDF form of the documentation requires that some or all of a tool like Windows/`MikTeX <https://miktex.org/>`_, Windows/`TeXlive <https://www.tug.org/texlive/>`_, Linux/`TeXlive <https://www.tug.org/texlive/>`_, and macOS/`MacTeX <https://tug.org/mactex/>`_ is available to the Sphinx build tool.
-Also check that you have ``latexmk`` available as this is required for creating the PDF form of the documentation.
+If this fails, please have a look at the following:
+
+Creating the PDF form of the documentation requires that some or all of a tool like Windows/`MikTeX <https://miktex.org/>`_, Windows/`TeXlive <https://www.tug.org/texlive/>`_, Linux/`TeXlive <https://www.tug.org/texlive/>`_, and macOS/`MacTeX <https://tug.org/mactex/>`_ is available to the Sphinx build tool.
+
+In addition, you'll need the following latex packages:
+
+`babel-greek`, `capt-of`, `cbfonts`, `courier`, `fncychap`, `fontspec`, `framed`, `greek-fontenc`, `helvetic`, `latexmk`, `needspace`, `tabulary`, `textgreek`, `titlesec`, `tocloft`, `varwidth`, `wrapfig`
+
+In particular, check that you have ``latexmk`` available as this is required for creating the PDF form of the documentation.
 Information on ``latexmk`` can be found at https://ctan.org/pkg/latexmk/.
 
+You may also need to install the ["GNU FreeFont" fonts](https://www.gnu.org/software/freefont/) `FreeSans`, `FreeSerif`, `FreeMono`.
```

Comment:
Double back ticks again. Also, add "and" before `FreeMono`.

Reply:
Thanks, just fixed this!

Reply:
?? Where?

Reply:
Didn't ou intend `FreeSans`, `FreeSerif` and `FreeMono` to be in code style? (Also add "and" before `FreeMono`.)